tools/dev/flash.sh

```diff
@@ -25,28 +25,35 @@ set -e
 : "${FLASH_SERIAL_PORT:=/dev/ttyUSB0}"
 : "${FLASH_OFFSET:=0x210000}"
 
-if [ -z "${IDF_PATH}" ]; then
-    echo "ERROR!  IDF_PATH must be set to root of IDF-SDK to pick up esptool."
+ESP_TOOL=`which esptool.py`
+if [ -z ${ESP_TOOL} ]; then
+    echo "ERROR!  esptool.py not found! IDF_PATH must be set to root of IDF-SDK to pick up esptool. Or install the standalone tool."
     exit -1
 fi
 
 usage() {
-    echo "Usage: $0 [-p <port>] [-b <baud>] <avm_file>"
+    echo "Usage: $ ${0} [-p <port>] [-b <baud>] [-o <offset>] [-l :: flash given .avm file to lib.avm partition] <avm_file>"
```

Comment:
What does `-l` mean?  Is that the offset if you are trying to flash the `lib.avm` partition?

May I suggest:
```
echo "Usage: $ ${0} [-p <port>] [-b <baud>] [-o <offset>] [-l] [-h] <avm_file>"
echo "where"
echo "    -p  <port> is the port connected to the device (default: /dev/ttyUSB0)"
echo "    -b <baud> is the baud rate (default: 115200)"
echo "    -o <offset> is the offset into flash to start writing the AVM file"
echo "    -l Use the flag (without offset) to flash the AtomVM core libraries"
echo "    -h print this help"
echo "    <avm-file> is the path to the AVM file"
echo ""
echo "Use the -l flag if you are flashing "